### PR TITLE
fix: migrate breadcrumbs to use theming parameters

### PR DESCRIPTION
**Pull request by droshev**
fixes: #397

### CODE REVIEW

File: src/breadcrumb.scss [hunk 1]
```diff
@@ -8,31 +8,39 @@
     .fd-breadcrump__link
 */
 
-$fd-breadcrumb-link-margin: fd-space(4);
+$fd-breadcrumb-margin-bottom: 0.5rem;
+$fd-breadbrum-spacing: 0.25rem;
+
+$fd-breadcrumb-font-size: var(--sapFontSize);
+$fd-breadcrumb-font-family: var(--sapFontFamily);
+$fd-breadcrumb-font-weight: normal;
```
Comment (InnaAtanasova) [inline src/breadcrumb.scss]: The `fd-reset()` is bringing the font-family and the font-weight

File: src/breadcrumb.scss [hunk 2]
```diff
@@ -8,31 +8,39 @@
     .fd-breadcrump__link
 */
 
-$fd-breadcrumb-link-margin: fd-space(4);
+$fd-breadcrumb-margin-bottom: 0.5rem;
+$fd-breadbrum-spacing: 0.25rem;
```
Comment (InnaAtanasova) [inline src/breadcrumb.scss]: typo - should be `$fd-breadcrumb-spacing`

File: src/breadcrumb.scss [hunk 3]
```diff
@@ -8,31 +8,39 @@
...
+
+$fd-breadcrumb-font-size: var(--sapFontSize);
+$fd-breadcrumb-font-family: var(--sapFontFamily);
+$fd-breadcrumb-font-weight: normal;
+
+$fd-breadcrum-separator-color: var(--sapContent_LabelColor);
+$fd-breadcrumb-item-color: var(--sapContent_LabelColor);
+$fd-breadbrum-focus-color: var(--sapContent_FocusColor);
```
Comment (InnaAtanasova) [inline src/breadcrumb.scss]: typo - should be `$fd-breadcrumb-....`

File: src/breadcrumb.scss [hunk 4]
```diff
@@ -83,12 +92,8 @@ $block: #{$fd-namespace}-breadcrumb;
...
-
-    &:focus {
-      outline-style: dotted;
-      outline-width: 1px;
-
-      @include fd-var-color("outline-color", fd-color-state("hover", "action"), --fd-color-action-focus);
-    }
+    font-size: $fd-breadcrumb-font-size;
+    font-family: $fd-breadcrumb-font-family;
+    font-weight: $fd-breadcrumb-font-weight;
```
Comment (InnaAtanasova) [inline src/breadcrumb.scss]: Since we have the reset I'm not sure we need this

**Comment by droshev** [comment]
@InnaAtanasova all your comments are addressed.

**Pull request closed without merging**